presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java

```diff
@@ -84,7 +85,7 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
     {
         JdbcTableHandle tableHandle = (JdbcTableHandle) table;
         ConnectorTableLayout layout = new ConnectorTableLayout(new JdbcTableLayoutHandle(tableHandle, constraint.getSummary()));
-        return ImmutableList.of(new ConnectorTableLayoutResult(layout, constraint.getSummary()));
+        return ImmutableList.of(new ConnectorTableLayoutResult(layout, TupleDomain.all()));
```

Comment:
But the Unenforced constraint shouldn't be `TupleDomain.none` right ? If the connector enforces all the domains then it should be `TupleDomain.all`. The base-jdbc connector ideally enforces all the domains so instead of passing the `constraint.getSummary()` we should pass `TupleDomain.all`

Reply:
Nevermind my previous comment. Please see: `com.facebook.presto.metadata.TableLayoutResult#computeEnforced`

I do not understand, why now it is better than `constraint.getSummary()`. The enforced constraint is calculated anyway.

Also can you provide a test that covers the case described: `currently all the jdbc connector have a filter expression on top of the table scan even though `TupleDomain` captures it and it is being enforced in the jdbc query so if we set the Unenforced constraint as `TupleDomain.all` presto need not do the filter operation once again`